_includes/configuration/media_alt_source.md

```diff
@@ -0,0 +1,22 @@
+### `media_alt_source`
+
+This options allows you disable the `Alternative source` input field in the media dialog.
```

Comment:
The media plugin also seems to have pronouns everywhere :cry:

Reply:
:disappointed: